mdx_linkify/mdx_linkify.py

```diff
@@ -5,23 +5,31 @@
 
 
 class LinkifyPostprocessor(Postprocessor):
-    def __init__(self, md, linkify_callbacks=[]):
+    def __init__(self, md, linkify_callbacks=[], linkify_parse_email=False):
         super(Postprocessor, self).__init__(md)
         self._callbacks = linkify_callbacks
+        self._parse_email = linkify_parse_email
 
     def run(self, text):
         text = bleach.linkify(text,
+                              parse_email=self._parse_email,
                               callbacks=self._callbacks,
                               skip_tags=['code'])
         return text
 
 
 class LinkifyExtension(Extension):
-    config = {'linkify_callbacks': [[], 'Callbacks to send to bleach.linkify']}
+    
+    def __init__(self, **kwargs):
+        self.config = {
+            'linkify_callbacks': [[], 'Callbacks to send to bleach.linkify'],
+            'linkify_parse_email': [False, 'Parse email addresses'],
+        }
+        super(LinkifyExtension, self).__init__(**kwargs)
 
     def extendMarkdown(self, md):
         md.postprocessors.register(
-            LinkifyPostprocessor(md, self.getConfig('linkify_callbacks')),
+            LinkifyPostprocessor(md, self.getConfig('linkify_callbacks'), self.getConfig('linkify_parse_email')),
```

Comment:
Should probably be explicit and pass `False` as the `default` kwarg to `getConfig` for the `linkify_parse_email` (it returns an empty string by default otherwise).
 ```suggestion
             LinkifyPostprocessor(md, self.getConfig('linkify_callbacks'), self.getConfig('linkify_parse_email', default=False)),
```

Reply:
Isn't that handled by the `self.config` at line 26?

Reply:
Oh yes, you are right!